polaris/ocean/tests/baroclinic_channel/rpe_test/analysis.py

```diff
@@ -123,14 +135,14 @@ def _plot(nx, ny, filename, nus, rpe):
         ax = axs[iCol]
         dis = ax.imshow(
             var_avg,
-            extent=[0, 160, 0, 500],
+            extent=[0, lx, 0, ly],
```

Comment:
I'm wondering if we should be defining the extent vs. letting it use auto-extent. Since it seems we are always showing the full domain, maybe it's better to let it plot everything just in case `lx` and `ly` are incorrect?

Reply:
That doesn't work with `imshow`.  We would need to get the extent from `xCell` and `yCell`.  I hate this little bit of code so much that I didn't want to get into it.  I'd rather replace it than edit it any further.  But I'm up for discussing it further.